src/common/api/authService.ts

```diff
@@ -0,0 +1,72 @@
+import { RootState } from '../../app/store';
+import { baseApi } from './index';
+
+const AUTH_SERVICE_URL = '/services/auth';
+
+interface TokenResponse {
+  created: number;
+  expires: number;
+  id: string;
+  name: string | null;
+  type: string;
+  user: string;
+  cachefor: number;
+}
+
+// Auth does not use JSONRpc, so we use queryFn to make custom queries
```

Comment:
When I start collections I will set up our default baseQuery so it can handle simple http calls alongside jsonRpc